internal/ingress/controller/store/store_test.go

```diff
@@ -121,631 +122,11 @@ func TestStore(t *testing.T) {
 		if err == nil {
 			t.Errorf("expected an error but none returned")
 		}
+
 		if svc != nil {
 			t.Errorf("expected an Ingres but none returned")
 		}
 	})
-
-	t.Run("should return one event for add, update and delete of ingress", func(t *testing.T) {
```

Comment:
why are you removing this whole tests without a replacement?

Reply:
Fixed